extensions/arc/src/ui/dashboards/postgres/postgresComputeAndStoragePage.ts

```diff
@@ -263,7 +263,6 @@ export class PostgresComputeAndStoragePage extends DashboardPage {
 		// Worker node count
 		this.workerBox = this.modelView.modelBuilder.inputBox().withProperties<azdata.InputBoxProperties>({
 			readOnly: false,
-			validationErrorMessage: loc.workerValidationErrorMessage,
```

Comment:
@nasc17 This was the only one that confused me - it seems like you want to do additional validation here to ensure that the value entered isn't below the current number of worker nodes. Is that something you're planning on adding later?

Reply:
The min is set when the model is refreshed and editWorkerNodeCount() is called.

Reply:
Ok, well the message is not going to be as clear then since it won't be telling the user WHY they can't go below the current node count. So if that's functionality that you want let me know and we can come up with something.